src/pages/Home.css

```diff
@@ -0,0 +1,60 @@
+#root {
+  background-color: rgb(190 221 211);
+  height: 100vh;
+  overflow-x: hidden;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.quran-details {
+  padding: 7%;
+}
+
+.heading {
+  background-color: rgb(140 205 186);
+}
+
+#myDiv .col {
+  box-shadow: 2px 2px 2px  lightblue;
+}
```

Comment:
- [ ] I like the fact that your app is unique, you chose a good API for the job. I have noticed that one some design requirements wasn't followed. The instructions for the design is that, we should follow everything from this [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), this includes colors, spaces, typograph, etc. Kindly change the background color to that of the given design.

![designing](https://user-images.githubusercontent.com/53124548/226105550-315bbd43-36d5-47c8-ad60-06bbc9515305.png)

## Yours

![color](https://user-images.githubusercontent.com/53124548/226105577-2c92a1b2-7f56-4dc7-8fa4-7aa6142d7e30.png)


## Original

![original1](https://user-images.githubusercontent.com/53124548/226106301-d48f1279-4890-450d-8634-b070a493d08d.png)

![orignal2](https://user-images.githubusercontent.com/53124548/226106316-d3541daf-066e-4e8b-81da-0448ae6ab6f4.png)